@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -102,4 +102,24 @@ export default {
       reset: "Tilbakestill tidsperspektiv",
     },
   },
+  DatePicker: {
+    close: "Lukk", // TODO: Consider making global
+    chooseDate: "Velg dato",
+    chooseDates: "Velg datoer",
+    chooseDateRange: "Velg start- og sluttdato",
+    chooseMonth: "Velg måned",
+    week: "Uke",
+    weekNumber: "Uke {week}",
+    selectWeekNumber: "Velg uke {week}",
+    month: "Måned",
+    nextMonth: "Gå til neste måned",
+    previousMonth: "Gå til forrige måned",
```

Comment:
Does it make sense to prefix them like this to better indicate its intention?
```suggestion
    goToNextMonth: "Gå til neste måned",
    goToPreviousMonth: "Gå til forrige måned",
```